guests_app/src/ViewModel/ItemViewModel.ts

```diff
@@ -0,0 +1,32 @@
+import {ItemModel} from '../Model/ItemModel';
+import Requests from '../Networking/requests';
+import Singleton from '../Singeltone';
+import {Item} from '../types';
+
+export class ItemsViewModel {
+	private itemsModel;
+	private requests: Requests;
+
+	constructor(requests: Requests) {
+		this.requests = requests;
+		this.itemsModel = new ItemModel();
+		this.syncItems();
```

Comment:
Just pointing out for future changes:
When putting the call to syncItems in the constructor and not in something external of the class comes with the problematic edge case of a failed network request. In the future its good to have a "try again" button in case the user didn't have a good internet connection.